bluesky/utils.py

```diff
@@ -1486,3 +1486,54 @@ def _rearrange_into_parallel_dicts(readings):
         data[key] = payload['value']
         timestamps[key] = payload['timestamp']
     return data, timestamps
+
+
+def plot_prune_fifo(bec, n, y, x):
```

Comment:
I'd prefer to have the arguments required to be specified with the kwarg names (put `*` after the `bec` argument). Also, having longer, self-explanatory names would greatly benefit the end users.

Reply:
None of these arguments are optional (kwargs) so wouldn't the `*` come after all of them?

But, to avoid having to provide the `bec` argument at all, this function might be better located as a method of BestEffortCallback.  Agree?

Reply:
Sorry.  I can't see any of these as kwargs.  Since no other args are parsed or accepted, there is nothing to change.

Reply:
I agree that making this a method is a good choice. Also, the new, more descriptive parameter names are much more clear.

To explain, I think @mrakitin is suggesting [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/) here so that `plot_prune_fifo` could _only_ be called by spelling out the parameter names.  Personally I think it would be OK to leave the arguments as the normal positional-or-keyword kind in this case, but that's a matter of taste.

Reply:
Right. Thanks for the reference, Dan. 

It would be much clearer when the args are spelled when the function is called, so we are not confused by the reverse order of x and y.